rasa/nlu/model.py

```diff
@@ -464,7 +465,10 @@ def parse(
         data = self.default_output_attributes()
         data[TEXT] = text
 
-        message = Message(data=data, time=timestamp)
+        output_properties = {TEXT_TOKENS}
+        message = Message(
+            data=data, time=timestamp, output_properties=output_properties
+        )
```

Comment:
Since we're not getting `TEXT_TOKENS` from `TOKENS_NAMES` anymore, should we change it like this?
```suggestion
        message = Message(
            data=data, time=timestamp, output_properties={TEXT_TOKENS}
        )
```